### PR TITLE
BAMOE ILMT Library for Quarkus

### PR DESCRIPTION
**Pull request by porcelli**
This PR address IBM/kie-roadmap#125

### CODE REVIEW

File: runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-common/pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+ <packaging>jar</packaging>
+
+  <artifactId>bamoe-ilmt-compliance-common</artifactId>
+  <name>Business Automation Manager Open Editions - ILMT Compliance Common</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-junit5</artifactId>
```
Comment (porcelli) [inline runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-common/pom.xml:20]: try use plain junit

File: runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-dmoe/pom.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.ibm.bamoe</groupId>
+    <artifactId>bamoe-ilmt-compliance</artifactId>
+    <version>9.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>bamoe-ilmt-compliance-dmoe</artifactId>
```
Comment (porcelli) [inline runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-dmoe/pom.xml:12]: use quarkus prefix

File: runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-pamoe/pom.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.ibm.bamoe</groupId>
+    <artifactId>bamoe-ilmt-compliance</artifactId>
+    <version>9.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>bamoe-ilmt-compliance-pamoe</artifactId>
```
Comment (porcelli) [inline runtime-libraries/bamoe-ilmt-compliance/bamoe-ilmt-compliance-pamoe/pom.xml:12]: use quarkus prefix

**Review by porcelli — COMMENTED** [review]
+ remove folder "bamoe" prefix, leave it only on pom artifact ids

**Review by baldimir — APPROVED** [review]

**Pull request merged**